src/editors/EditorContainer.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { getConfig } from '@edx/frontend-platform';
+
+import EditorPage from './EditorPage';
+
+interface Props {
+  /** Course ID or Library ID */
+  learningContextId: string;
+  /** Event handler for when user cancels out of the editor page */
+  onClose?: () => void;
```

Comment:
This issue is with the underlying editor, but passing `onClose={undefined}` here actually crashes the `TextEditor`.

Reply:
Tried to address this with 9e62bc05d79887c185186d8ce2124c5974e52eaf